_sass/brand.scss

```diff
@@ -256,45 +347,93 @@
         margin-left: 0;
     }
 
-    .content-body-section img {
+    .content-body-section {
+        width: 20rem;
+    }
+
+    .content-body-section .show {
         display: none;
     }
 
     .content-body-section .mobile-image {
         display: block;
-        width: 100%;
+        width: 20rem;
     }
 
     .logo-container {
-    flex-direction: column;
-    align-items: center;
-    }
-    
-    .logo-container img{
-        display: block;
+        flex-direction: column;
+        align-items: center;
+        padding: 0;
     }
 
-    .logo-container-img2 {
-        width: 100%;
-        border-top: 2px solid #3d4f57;
-        border-left: none;
-        padding: 20px 0;
+    .logo-container img {
+        display: block;
     }
 
     .logo-container-img1 {
         border-right: none;
+        border-bottom: 2px solid #3d4f57;
         width: 50% !important;
         height: 70%;
         padding: 2rem 0px;
         aspect-ratio: 1/1;
     }
+
+    .meshsync-logo-container-img {
+        border-left: none;
+        width: 50% !important;
+        height: 70%;
+        padding: 2rem 0px;
+        aspect-ratio: 1/1;
+        border-top: 2px solid #3d4f57;
+    }
+
+    .color-palette {
+        display: flex;
+        flex-wrap: wrap;
+        justify-content: center;
+        align-items: center;
+        flex-direction: row;
+        margin: 0;
+        padding: 0;
+    }
+
+    .color-box {
+        width: 140px;
+        height: 140px;
```

Comment:
Let's please avoid writing "px" to the extent possible @psankhe28.
can be replaced by "rem", "em" etc.

Reply:
It was discussed with Rex and he suggested to use 140px